codeHintsInstall.php

```diff
@@ -0,0 +1,295 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: Vladimir.Malinovskiy
+ * Date: 2/7/2017
+ * Time: 11:11 PM
+ */
+
+$docText =
+'/**
+ * Api client class
+ *
+ * The class is used to instantiate api client
+ *
+ * @package    Callfire
+ * @subpackage Api
+ * @author     Justin Martin <jmart@callfire.com>
+ */';
+
+$callFireSwaggerUrl = 'https://www.callfire.com/v2/api-docs/swagger.json';
+$spec = file_get_contents($callFireSwaggerUrl);
+$decodedSpec = json_decode($spec, true);
+$definitions = $decodedSpec['definitions'];
+$paths = $decodedSpec['paths'];
+foreach ($paths as $apiBunch) {
+    foreach ($apiBunch as $apiInstance) {
+        $docText .= "\n\n".loadDataToFile($apiInstance);
+    }
+}
+
+//UPDATING Client.php class to include documentation
+$file = './src/CallFire/Api/Client.php';
```

Comment:
This path will resolve differently when the client is installed as a dependency.